components/Input/input.css

```diff
@@ -0,0 +1,77 @@
+/* global css file for fonts and icons  */
+@import '/css/main.css';
+
+/* input class main*/
+.input-types {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    margin: 1rem;
+    padding: 1rem;
+    height: 20rem;
+    width: 30rem;
+    background-color: var(--background-universal-color);
+
+}
+/* global html5 input element */
+input {
+    width: 20rem;
+    height: 2rem;
+    margin: 0.3rem;
+    padding: 0.2rem;
+    border-radius: 0.4rem;
+    color: white;
+}
+/* html5 input type submit button */
+input[type='submit'] {
+    height: 2rem;
+    width: 5rem;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    cursor: pointer;
+    font-size: 1.2rem;
+    font-weight: bold;
+    color: white;
+   
+}
+
+
+
+
+input + span {
+    position: relative;
+  }
+  
+  input + span::before {
+    right: -20px;
+    top: 5px;
+  }
+  
+  input:invalid {
+    border: 2px solid red;
+    background-color: red;
+  }
+  input:valid{
+      border: 2px solid greenyellow;
+      background-color: green;
+  }
+  
+  
+  input:invalid + span::before {
+    content: '✖';
+    color: red;
+    font-size: 1.5rem;
+  }
+  
+  input:valid + span::before {
+    content: '✓';
+    color: green;
+    font-size: 1.5rem;
+  }
+  
+
+
```

Comment:
You have used global tags in this file, you have to change it otherwise you will get in problems, give them class names and than use them